src/wallet/wallet.cpp

```diff
@@ -2644,7 +2644,7 @@ bool CWallet::DisplayAddress(const CTxDestination& dest)
         ExternalSigner signer = ExternalSignerScriptPubKeyMan::GetExternalSigner();
         return signer_spk_man->DisplayAddress(dest, signer);
     }
-    return false;
+    return util::Error{_("No external signer ScriptPubKeyManager")};
```

Comment:
In 5536ace894e67322161a649f998265f461fbc8ad: I think we can improve the "No external signer ScriptPubKeyManager" message, it doesn't sound intuitive. First time I tested it prior reading the code, I got this error and I thought I didn't have any external signer, so I checked it with `enumerate`.

Perhaps: "There is no ScriptPubKeyManager for this address".

Reply:
Will consider new wording if I need to touch / rebase.

Reply:
Done